src/components/skills-quiz/SkillsQuiz.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
Shouldn't the name of this file be `index.jsx`?

Reply:
Actually, this file does not contain any `jsx` syntax, that is why `.jsx` extension is not needed. Also, I used this approach because same conventions is used at all other places in this repo.